libs/ivrAction/say.js

```diff
@@ -1,50 +1,52 @@
 module.exports = class SayAction {
-    constructor(sayData,nextLayer,language) {
-//call.Say({sayData:[{"Number" : "113"}],NextLayer:13 ,Language :"EN" })
```

Comment:
Lost comments?

Reply:
It's an example in code. I really don't know if we need it, but it really help me to understand how it works.